src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/RoatpGatewayPageViewModel.cs

```diff
@@ -13,27 +13,19 @@ public class RoatpGatewayPageViewModel
         public string Status { get; set; }
 
         public string Ukprn { get; set; }
-        public string UkrlpLegalName { get; set; }
```

Comment:
You've removed UkrlpLegalName but I haven't seen this placed in whatever viewmodels & pages need to check UKRLP stuff

Reply:
We need to use only ApplyLegalName as common property in RoatpGatewayPageViewModel. And it is the only one populated by PopulatePageCommonDetails.
If someone needs UkrlpLegalName in their particular ViewModel, they can deal with it in their implementation.